### PR TITLE
[spike] Add `View > Team Explorer` button to `GitHub` pane command bar

### LINKED ISSUE #1558: Menu Buttons for PR Pane Conflict with Every Other Pan
I recognize that the PR pane is GitHub, versus the others (home and changes in this example) are Team Explorer, however I have noticed a jarring experience when I'm going between PRs and Changes because there is no quick and easy way to get back to the Team Explorer home (where I then have access to the rest of my actions). 

__Team Explorer Home__:
screen shot 2018-03-21 at 11 14 54 am

__Changes__:
screen shot 2018-03-21 at 11 15 00 am
I can get back to the Team Explorer home by clicking the Home icon.

__PRs__:
screen shot 2018-03-21 at 11 15 07 am
I can go back to the list view of PRs or open a browser window into GitHub, but I can't get back to the Team Explorer home. To get there, I have to go down to the bottom and click on the Team Explorer tab. 

This may not be a major issue for most people, but Im' curious about the design decision here (i.e. not having a button up there to go home). I've found myself accidentally opening the GitHub browser window multiple times when trying to just get back to the Team Explorer Home pane. 

_Curious_

### PR DESCRIPTION
**Pull request by jcansdale**
#### This PR

- Add the `View > Team Explorer` button the the `GitHub` pane's command bar

image

- ~~Use standard `Backwards`, `Forwards`, `OpenWebSite`, `Refresh` and `StatusHelp` monikers for `GitHub` command bar (found in `KnownImageIds.vsct`)~~ *Moved to #1876*

- Add view `GitHub` pane button to `Team Explorer` command bar

image

Mentioned by @sguthals in #1558 

> I can go back to the list view of PRs or open a browser window into GitHub, but I can't get back to the Team Explorer home. To get there, I have to go down to the bottom and click on the Team Explorer tab.

#### Todo

- An easy way to navigate back to the `GitHub` pane

When you try this functionality, the question quickly becomes; how do I get back to the `GitHub` pane.

I don't know if it's possible to add buttons to the `Team Explorer` command bar. If it was, the symmetry would make it feel pretty natural and discoverable. It would also let users easily return to `GitHub` from the `Team Explorer - Changes` view (which is a common workflow).

An alternative would be to allow users to click on the icon below `GitHub` to return to the `GitHub` pane. This wouldn't be available on the `Team Explorer - Changes` view, so users would need to navigate back via `Home`.

image

- There are built in monikers for `Forwards`, `Backwards` and (maybe) `Home` that we could use.

This would make it consistent wit the `Team Explorer` and `Solution Explorer` tool bars:
image

#### Related

- Team Explorer commands and command bar layout
https://gist.github.com/jcansdale/619de883f37b88674393673524216474

### CODE REVIEW

**Comment by jcansdale** [comment]
Here is the Backwards

**Comment by sguthals** [comment]
Thanks @jcansdale for this! I want to punt it to our first 3+1 week to get more feedback. I like this approach _if_ it's the problem we want to solve. We may want to be solving a different problem though - making it less streamlined.

File: src/GitHub.VisualStudio/GitHub.VisualStudio.vsct [hunk 1]
```diff
@@ -3,6 +3,9 @@
   xmlns="http://schemas.microsoft.com/VisualStudio/2005-10-18/CommandTable"
   xmlns:xs="http://www.w3.org/2001/XMLSchema">
 
+  <!-- Import the standard ImageCatalogGuid monikers -->
```
Comment (sguthals) [inline src/GitHub.VisualStudio/GitHub.VisualStudio.vsct]: LGTM

File: src/GitHub.VisualStudio/GitHub.VisualStudio.vsct [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
+  <Include href="KnownImageIds.vsct"/>
+
```
Comment (sguthals) [inline src/GitHub.VisualStudio/GitHub.VisualStudio.vsct]: Starting review

**Comment by jcansdale** [comment]
I've extracted the following commits to separate PRs (they shouldn't have been included in this PR):

- #1877 Use OpenWebSite image for View/Open on GitHub 
- #1876 Use standard image monikers for GitHub pane

**Pull request closed without merging**